xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/META-INF/components.txt

```diff
@@ -146,6 +146,7 @@ com.xpn.xwiki.internal.template.TemplateListener
 com.xpn.xwiki.internal.template.VelocityTemplateEvaluator
 com.xpn.xwiki.internal.query.ConfiguredQueryExecutorProvider
 com.xpn.xwiki.internal.query.CurrentLanguageQueryFilter
+500:com.xpn.xwiki.internal.redirection.RedirectClassRedirectionFilter
```

Comment:
Why 500 ?

Reply:
Arbitrary value to make it executed first and decrease the chances to see regression caused by other components. But might be over-engineered indeed.